recipe/meta.yaml

```diff
@@ -24,11 +24,11 @@ requirements:
     - python
     - setuptools
     - blas 1.1 {{ variant }}
-    - openblas 0.2.19|0.2.19.*
+    - openblas 0.2.20  # [linux]
   run:
     - python
     - blas 1.1 {{ variant }}
-    - openblas 0.2.19|0.2.19.*
+    - openblas >=0.2.15  # [linux]
```

Comment:
Yep. I'll investigate that later but it won't affect us. Virtually everything that links to `openblas`, like `scipy`,  needs `numpy >1.7` anyways.